cypress-tests/cypress/support/commands.js

```diff
@@ -86,6 +86,57 @@ Cypress.Commands.add("apiKeyCreateTest", (apiKeyCreateBody, globalState) => {
   });
 });
 
+Cypress.Commands.add(
+  "createNamedConnectorCallTest",
+  (
+    createConnectorBody,
+    payment_methods_enabled,
+    globalState,
+    connectorName
+  ) => {
+    const merchantId = globalState.get("merchantId");
+    createConnectorBody.connector_name = connectorName;
+    // createConnectorBody.connector_name = globalState.get("connectorId");
+    createConnectorBody.payment_methods_enabled = payment_methods_enabled;
+    // readFile is used to read the contents of the file and it always returns a promise ([Object Object]) due to its asynchronous nature
+    // it is best to use then() to handle the response within the same block of code
+    cy.readFile(globalState.get("connectorAuthFilePath")).then(
+      (jsonContent) => {
+        const authDetails = getValueByKey(
+          JSON.stringify(jsonContent),
+          globalState.get("connectorId")
+        );
+        console.log(`>>>>>>>>>>>>>>>>>>>>>>>>auth "${authDetails}"`);
+        createConnectorBody.connector_account_details = authDetails;
+        cy.request({
+          method: "POST",
+          url: `${globalState.get("baseUrl")}/account/${merchantId}/connectors`,
+          headers: {
+            "Content-Type": "application/json",
+            Accept: "application/json",
+            "api-key": globalState.get("adminApiKey"),
+          },
+          body: createConnectorBody,
+          failOnStatusCode: false,
+        }).then((response) => {
+          logRequestId(response.headers["x-request-id"]);
+
+          if (response.status === 200) {
+            expect(globalState.get("connectorId")).to.equal(
+              response.body.connector_name
+            );
+          } else {
+            cy.task(
+              "cli_log",
+              "response status -> " + JSON.stringify(response.status)
+            );
+          }
+        });
+      }
+    );
+  }
+);
+
```

Comment:
Why do we need this? Can you modify the existing `connectorCreateCall` instead?

Reply:
Not modifying the above as there will be two functions :
1. Where the Connector can be directly derived from the 'Connector-id'.
2. Where the Connector can be derived using name.
However we can modify both functions, into one by having a check on param. If provided get the connector using name else get from ID. This can be taken up as good-first issue.

Reply:
My intention was to have a single function to make a connector create call. That way we would be having more control as we'll be passing `connectorId` directly from configs and handle payouts scenario as well in one.

Reply:
This can be a medium amount of refactor(considering the fact that we will have to refactor on all the places wherever the `getConnector` is being called ), which if you want I can take up in next PR, as this one needs to go today and this one consists of a lot of logic that can cause issues already, so want this one to be separate.

Reply:
Agreed. To my knowledge, it is going to be a small refactor to `connectorCreateCall`. Can be taken up as a separate PR and I think I'll be assigned.